lib/source-map-cache.js

```diff
@@ -20,16 +20,31 @@ SourceMapCache.prototype.applySourceMaps = function (coverage) {
   var mappedCoverage = _.cloneDeep(coverage)
 
   Object.keys(coverage).forEach(function (key) {
-    if (_this.cache[key]) {
-      _this._rewriteStatements(mappedCoverage[key], _this.cache[key])
-      _this._rewriteFunctions(mappedCoverage[key], _this.cache[key])
-      _this._rewriteBranches(mappedCoverage[key], _this.cache[key])
+    var sourceMap = _this.cache[key]
+    if (!sourceMap) {
+      return
     }
+
+    var fileCoverage = mappedCoverage[key]
+    _this._rewritePath(mappedCoverage, fileCoverage, sourceMap)
```

Comment:
@cades could you open a new issue for this? Any sample code allowing us to reproduce the error locally would be appreciated. Thanks.